### DISCUSSION TITLE
[Question]: Is is possible to see where motion was triggered?

### DISCUSSION THREAD

**Discussion by jamesonuk** [blakeblackshear/frigate#12637] [Ask A Question]
### What is your question:

I have some recordings where I can't figure out what triggered them to be even considered for object detection.

The worst one is a camera I have in one of the kids rooms that every so often will start a recording and detect that he is in bed (or often that one of the toys in his room is a person...) and then all be all close down again only to have the same happen again several times over the night.

My recordings are set to retain motion so I think I am right in saying that each of the events I see must have been triggered by motion and the detection kicks in, finds a person and then keeps the video.   The issue is that with my eyes I cannot see any motion (changes in light etc.) so I can't identify if there is a particular spot I need to mask off or something I need to adjust.

Is there anyway I can see the details of where it detected motion in a similar way you can see the person detected / stationery / left boxes when viewing events?

**Comment by NickM-27** [comment]
In 0.13 you can use the buttons that are on top of the clip

In 0.14 there is no equivalent view, we have plans to add a page dedicated for this in the future. 

You can always create a test camera on one of the clips, using https://docs.frigate.video/development/contributing#2-modify-your-local-config-file-for-testing

**Reply by jamesonuk** [reply]
> In 0.13 you can use the buttons that are on top of the clip

That only shows when the objects were detected through?   There must have been motion (potentially somewhere else in the image) that triggered it to the object detection?

Similarly, as I have recordings set to retain motion, I have some recordings that don't generate an event and would really like to be able to work out what triggered those too.

**Reply by NickM-27** [reply]
you'll need to re-run the recordings to get motion info

**Comment by giuliodali** [comment]
Hi, just wondering if there is an update to this? I've tuned to what looks good in debug view but things are still triggering and it would be good to see on the recorded clip what region the motion was detected and also where it thinks there  was an object. This is retrospective after the clip has recorded.
